src/emm/services/git_service.py

```diff
@@ -147,4 +147,4 @@ def _determine_directory(directory: Optional[Path] = None) -> Path:
             return Path(local.cwd)
         elif not directory.is_absolute():
             return Path(local.cwd / directory)
-        return directory
+        return Path(directory)
```

Comment:
Why is this changing? It seems wrong. The directory being passed in should already be a path.

Reply:
Change it back. I was trying to test if this is the case test failed on Windows, it proved not relevant to this return value.